docker-compose/docker-compose-injistack/certify_init.sql

```diff
@@ -0,0 +1,166 @@
+);
+
+CREATE TABLE certify.key_store(
+                                  id character varying(36) NOT NULL,
+                                  master_key character varying(36) NOT NULL,
+                                  private_key character varying(2500) NOT NULL,
+                                  certificate_data character varying NOT NULL,
+                                  cr_by character varying(256) NOT NULL,
+                                  cr_dtimes timestamp NOT NULL,
+                                  upd_by character varying(256),
+                                  upd_dtimes timestamp,
+                                  is_deleted boolean DEFAULT FALSE,
+                                  del_dtimes timestamp,
+                                  CONSTRAINT pk_keystr_id PRIMARY KEY (id)
+);
+
+CREATE TABLE certify.svg_template (
+                                    id UUID NOT NULL,
+                                    template VARCHAR NOT NULL,
+                                    cr_dtimes timestamp NOT NULL,
+                                    upd_dtimes timestamp,
+                                    CONSTRAINT pk_svgtmp_id PRIMARY KEY (id)
+);
+
+CREATE TABLE certify.template_data(
+                                    context character varying(1024) NOT NULL,
+                                    credential_type character varying(512) NOT NULL,
+                                    template VARCHAR NOT NULL,
+                                    cr_dtimes timestamp NOT NULL default now(),
+                                    upd_dtimes timestamp,
+                                    CONSTRAINT pk_template PRIMARY KEY (context, credential_type)
+);
+
+INSERT INTO certify.template_data (context, credential_type, template, cr_dtimes, upd_dtimes) VALUES ('https://vharsh.github.io/DID/mock-context.json,https://www.w3.org/2018/credentials/v1', 'MockVerifiableCredential,VerifiableCredential', '{
+    "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://vharsh.github.io/DID/mock-context.json"],
+    "issuer": "${issuer}",
+    "type": ["VerifiableCredential", "MockVerifiableCredential"],
+    "issuanceDate": "${validFrom}",
+    "expirationDate": "${validUntil}",
+    "credentialSubject": {
+        "gender": ${gender},
+        "postalCode": ${postalCode},
+        "fullName": ${fullName},
+        "dateOfBirth": "${dateOfBirth}",
+        "province": ${province},
+        "phone": "${phone}",
+        "addressLine1": ${addressLine1},
+        "region": ${region},
+        "vcVer": "${vcVer}",
+        "UIN": ${UIN},
+        "email": "${email}",
+        "face": "${face}"
+    }
+}', '2024-10-22 17:08:17.826851', NULL);
+INSERT INTO certify.template_data (context, credential_type, template, cr_dtimes, upd_dtimes) VALUES ('https://vharsh.github.io/DID/mock-context.json,https://www.w3.org/ns/credentials/v2', 'MockVerifiableCredential,VerifiableCredential', '{
```

Comment:
can we move context to mosip repo instead of using personal repos?
I am referring to this https://vharsh.github.io/DID/mock-context.json

Reply:
Anyone changing the Usecase will have to update this to point to their repository